compiler/bcq-tools/generate_bcq_output_arrays

```diff
@@ -83,20 +95,40 @@ def print_output_arrays(flags):
 
 
 def find_bcq_version(flags):
-    pass
+    """
+    If BCQ metadata exists, BCQ version is in the second element.
+    Return -1 when the metadata is not found.
+    """
+    graph = load_graph(flags.input_path)
+    graph_def = graph.as_graph_def()
+    for node in graph_def.node:
+        if node.op == "Const" and "one_compiler/bcqinfo_one_metadata" in node.name:
+            metadata_tensor = tf.make_ndarray(node.attr["value"].tensor)
+            return metadata_tensor[1]
+    return -1
 
 
 def print_bcqinfo_output_arrays_v1(flags):
     pass
 
 
 def print_bcq_output_arrays(flags):
-    version = find_bcq_version(flags)
+    program_version = 1
+    model_version = find_bcq_version(flags)
     
-    if version == 1:
+    if model_version == 1:
```

Comment:
How about `model_version == program_version` ?

Reply:
If `program_version` is 3, it means that it can support `model_version` from 1 to 3.
Therefore we cannot use like that :(

Reply:
@llFreetimell , merge? or would you like to separate `print_bcq_output_arrays`  from this PR?

Reply:
@seanshpark Merge plz :)

Reply:
> If program_version is 3, it means that it can support model_version from 1 to 3.

Then `if model_version >= 1 and model_version <=  program_version:` looks better to me :)